pkg/config/cos.go

```diff
@@ -446,3 +462,31 @@ func getAddStaticDNSServersCmd(servers []string) string {
 func (c *HarvesterConfig) ToCosInstallEnv() ([]string, error) {
 	return ToEnv("COS_INSTALL_", c.Install)
 }
+
+// Returns Rancherd bootstrap resources
+// map: fileName -> fileContent
+func genBootstrapResources(config *HarvesterConfig) (map[string]string, error) {
+	bootstrapConfs := make(map[string]string, 4)
+
+	for _, templateName := range []string{
+		"10-harvester.yaml",
+		"11-monitoring-crd.yaml",
+		"13-monitoring.yaml",
+	} {
+		rendered, err := render("rancherd-"+templateName, config)
+		if err != nil {
+			return nil, err
+		}
+
+		bootstrapConfs[templateName] = rendered
+	}
+	// It's not a template but I still put it here for consistency
```

Comment:
Normally it's no harm to render a file without actions in it.
But given that `12-monitoring-dashboard.yaml` contains a long JSON text and there are also `{{ }}` text in it,  +1 to treat the file separately.